### PR TITLE
Add missing sites to Complex Media

### PR DESCRIPTION
**Pull request by fabriceTOUSSAINT**

### CODE REVIEW

**Comment by fmarier** [comment]
Fabrice, could you please [file a bug on Bugzilla](https://bugzilla.mozilla.org/enter_bug.cgi?product=Firefox&component=Tracking%20Protection) describing the underlying problem (broken site?) that this omission from the list creates?

Then we'll reference that bug in this commit as justification for the change.

**Comment by fabriceTOUSSAINT** [comment]
Hi @fmarier, I just filled out a Bugzilla report ( https://bugzilla.mozilla.org/show_bug.cgi?id=1315740 )

File: disconnect-entitylist.json [hunk 1]
```diff
@@ -2354,11 +2354,21 @@
...
+            "solecollector.com",
+            "pigeonsandplanes.com",
+            "theridechannel.com",
+            "firstwefeast.com"
         ],
         "resources": [
             "complexmedianetwork.com",
-            "complex.com"
+            "complex.com",
+            "collider.com",
```
Comment (fmarier) [inline disconnect-entitylist.json]: As per [bug 1315740](https://bugzilla.mozilla.org/show_bug.cgi?id=1315740) these are top-level properties, not resources that other sites pull in. Also, they're not on the Disconnect blocklist, so they only need to be in the `properties` section, not under `resources`.

**Comment by fmarier** [comment]
Thanks for filing the bug.

When you fix the `resources` issue I noted abobe, please add a link to https://bugzilla.mozilla.org/show_bug.cgi?id=1315740 in the commit message.

**Comment by fmarier** [comment]
I've looked into the domain ownership in https://bugzilla.mozilla.org/show_bug.cgi?id=1315740 and so this looks good to me.

@groovecoder Can you take a look too and give your r+ before I squash these commits and fix the commit message to include the bugzilla bug?

**Pull request merged**